src/DependencyInjection/LeezyPheanstalkExtension.php

```diff
@@ -22,10 +25,11 @@ class LeezyPheanstalkExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+        $loader->load('commands.xml');
 
         if (!$config['enabled']) {
```

Comment:
Why is `enabled` even an option? Doesn't make sense as the bundle doesn't do anything else, really.